app/views/obituaries/_form.html.erb

```diff
@@ -1,25 +1,51 @@
-<%= form_for @obituary do |f| %>
-
-  <%= f.label :first_name %>
-  <%= f.text_field :first_name  %>
-
-  <%= f.label :last_name %>
-  <%= f.text_field :last_name %>
-
-  <%= f.label :age %>
-  <%= f.number_field :age, min: 0, max: 150 %>
-
-  <%= f.label :city %>
-  <%= f.text_field :city %>
-
-  <%= f.label :state %>
-  <%= f.text_field :state %>
-
-  <%= f.label :description %>
-  <%= f.text_field :description, :size => 150 %>
-
-  <%= f.label :image_url %>
-  <%= f.text_field :image_url %>
-
-  <%= f.submit %>
-<% end %>
+<div class="container">
```

Comment:
I tried to make this page as Ada compliant/accessible as I could, but it definitely needs some work still.

Reply:
That is so great you looked that up! Please teach us!